bin/github-changelog-generator

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env php
+<?php
+
+require_once '../vendor/autoload.php';
+
+// define CLI arguments
+$cli = new Commando\Command();
```

Comment:
I was pretty hesitant at first at adding another library, but this has support for built in error handling/exit codes, support for arguments without a name, aliases, even a help menu; and eventually, I can easily add support for templates and label mappings (`--template` and `--labels`). These were all features I was going to add to the original, but I currently don't have the time to implement all this at the moment. This library was a shortcut until I can get around to writing an implementation that doesn't require a third party library (in other words, this dependency will be removed in a later version).